content/300-accelerate/250-connection-pooling.mdx

```diff
@@ -4,39 +4,56 @@ metaTitle: 'Accelerate: Connection Pooling'
+
+In some cases, however, it may be beneficial to further configure connection pooling in order to optimize the performance of your application.
 
-### Default connection pool size
+### Connection Pooling in Accelerate
 
-Accelerate has a default `connection_limit` set to `10`.
+By default, Accelerate has a default connection limit of `10`. In most cases, this connection limit is sufficient. However, in high traffic or compute environments, you may need to [increase this value](#configuring-the-connection-pool-size). For the [Pro plan](https://prisma.io/pricing) there is a maximum connection limit of `100`. For [Business plan](https://prisma.io/pricing) users the maximum is `1000`.
 
 ### Configuring the connection pool size
 
-The connection pool size can be configured via the _database connection string_ in your Platform project (_not_ the Accelerate connection string in your Prisma schema) via the `connection_limit` parameter. For example, set a connection pool size of `10` like this: `postgresql://USER:PASSWORD@HOST:PORT/DATABASE?connection_limit=10`.
+The connection pool size can be configured in your Platform project via the `Connection limit` setting. For example, set a connection pool size of `10` like this:
+
+![Update database connection string in Accelerate](/img/accelerate/accelerate-update-database-connection-string.png)
 
-If nothing is configured, a default value will be used.
+If nothing is configured, the default value of `10` will be used.
 
 To change the size of the connection pool:
 1. Open the [Console](https://console.prisma.io/login).
 1. Select the _project_ in which you're using Accelerate. You might need to select a different _workspace_ if you can't see your project.
 1. Select the _environment_ where you want to configure Accelerate's connection pool size.
-1. Update the connection string by appending the `connection_limit` argument. 
+1. Update the connection limit via the `Connection limit` setting.
 
 ![Update database connection string in Accelerate](/img/accelerate/accelerate-update-database-connection-string.png)
 
+#### Autoscaling
+
+Autoscaling allows Accelerate to allocate resources to scale up or down based on the traffic to your application. Specifically, Accelerate will begin allocating new resources when you have consistent usage greater than 90% of your currently alloted connection limit. If traffic continues to increase, Accelerate will scale up further. If traffic decreases, Accelerate will scale down.
+
+Autoscaling is enabled when the Accelerate connection limit is set higher than `10`. Autoscaling is not available on the [Starter plan](https://prisma.io/pricing).
```

Comment:
This is confusing. Below in the table it says starter is 10. But here it says its not available. But overall this section reads much better is is laid out nicely